proxima-chapter-1.md

```diff
@@ -228,92 +230,107 @@ Then, let's proceed to prepare the environment for subsequent steps like:
+3. Generate a Certificate Signing Request
+    - openssl req -new -key server.key -out server.csr
+4. Sign the Certificate Signing Request
+    - openssl x509 -req -days 365 -in server.csr -CA ca.crt -CAkey ca.key -set_serial 01 -out server.crt
+5. Verify that every bit is ok
+    
+    ```sh
+    $ openssl rsa -noout -text -in server.key
+    $ openssl req -noout -text -in server.csr
+    $ openssl rsa -noout -text -in ca.key
+    $ openssl rsa -noout -text -in ca.key
+    ```
+
+6. Make a version of the server key that doesn't need the password
+    
+    ```
+    openssl rsa -in server.key -out server.key.insecure
+    mv server.key server.key.secure
+    mv server.key.insecure server.key
+    ```
+
+7. Edit the file /usr/lib/python3.6/site-packages/certifi/cacert.pem adding the ca.crt file content (PEM public Key)
 
 Be sure to put (and mantain!) the appropriate files in a certs directory under /etc/mosquitto and have this directory saved where you want to perform some tests, like: 
 
-`[opc@proximaserver ~]$ mosquitto_pub -t 'edge/device/machines' -h <IP
+```sh
+$ mosquitto_pub -t 'edge/device/machines' -h <IP
 Address of OCI compute> -u <username> -P <password> -p <your port> --
 insecure --cafile certs/ca.crt --cert certs/server.crt --key
-certs/server.key -m '{"Developers Developers Developers Developers!"}'`
+certs/server.key -m '{"Developers Developers Developers Developers!"}'
+```
 
 We won't configure bridging (the dialog between Edge Mosquitto and Cloud Mosquitto) for now, that will be done in the second part of our prolix boomer babbling.
 
 #### Get the OCI SDK for Python and Install Libraries
 
-Well, it's time to perform some moves on the chessboard:
+Well, it's time to perform some moves on the chessboard. Clone the git repository from GitHub:
 
-- Clone the git repository from GitHub: `git clone https://github.com/oracle/oci-python-sdk.git`.
+```sh
+git clone https://github.com/oracle/oci-python-sdk.git
+```
 
 This repository contains a wealth of source code and examples, and is regularly updated to reflect the evolution of OCI APIs. Then,
 
-- Install the Python SDK:  `pip install oci`
-- Install the Paho Library: `pip install paho-mqtt`
+1. Install the Python SDK:  `pip install oci`
+2. Install the Paho Library: `pip install paho-mqtt`
 
 and we're ready to write and execute a little python module that will grab sensors messages from the Cloud-side Mosquitto and route the messages to OCI streaming, which in turn will be the source connection for Golden Gate Stream Analytics. To do that, we'll grab the example named stream_example.py and taylor it to our needs. The module is available here (NOTE: insert the GitHub link, open in a new window/tab).
 
 Here are some snippets.
 
-Import the **paho** library enabling the subscription to MQTT topics managed by the Cloud-side Mosquitto.
+Import the `paho` library enabling the subscription to MQTT topics managed by the Cloud-side Mosquitto.
```

Comment:
Any time you're mentioning a command, UNIX tool, language library, etc., make it a code span. Anything you would type on a command line or in an IDE should be code.